### DISCUSSION TITLE
Is there a time window or method to go back to stable without downgrading?

### DISCUSSION THREAD

**Discussion by neocult-de** [nextcloud/all-in-one#3547] [Questions] [answered]
Hi everyone,

have someone experience with going from beta back to stable? I know I can just switch the channel from nextcloud/all-in-one:latest to nextcloud/all-in-one:beta for the master container, but doing so the AIO web interface recommends an "Upgrade" and downgrades the containers to the older version of the stable channel. This seems hardly a recommended procedure and behavior to me.  

Because of open connections to postgres issue and Talk connectivity for NATed Clients, I have updates to the beta Channel (currently AIO 7.4.1), which resolved the issues. But is it possible to either not update (How about automatic updates at night?), somehow fix the version to go back to stable later? Switching channels when necessary is no issue, but being stuck in beta might bring the same issue later on. 

Any ideas how and in what time window it could be possible to safely switch over to the stable channel (e.g. stay on 7.4.1 until stable release)?

**Comment by szaimen** [comment, marked as answer]
> 7.4.1 until stable release

7.4.1 stable is already released so you should be able to switch without any problem now

**Comment by neocult-de** [comment]
Thanks a lot for the quick answer. The switch back to latest now works, by it updating itself using the prior image or by pulling the latest manually and then starting it as usual.

Will there usually be the time windows where the beta becomes the release and both Nextcloud AIO channels share the same version in the latest release?

**Reply by szaimen** [reply]
Usually switchting between beta and latest should work any time. Only fulltextsearch might be a problem.

**Reply by neocult-de** [reply]
Okay had exactly that issue. Was worried switching back to lower versions of Nextcloud or other containers might corrupt data. Thanks again.
